content/resource/models.md

```diff
@@ -0,0 +1,40 @@
+---
+title: models
+
+aliases:
+  - /models.html
+
+---
+
+A collection of Alloy models to peruse...
+
+<!--more-->
+
+### Alloy Citations
+
+   |     
+ --------- | ------------- 
+ [Google scholar](http://scholar.google.com/scholar?q=alloy+language+OR+model+-hysteresis+-metal+-steel+-thermal+-titanium+-surface+-magnetic+-weld+-anneal&hl=en&num=100&as_sdt=1,22&as_ylo=1997&as_subj=eng) | A sample of over 1200 papers discussing Alloy
+ [case studies]({{% ref case-studies %}}) | Selected papers describing applications of Alloy 
+ [translations](citations/language-translations.html) | Papers on translating a dozen other languages into Alloy 
+ [theses](citations/theses.html) | Theses about Alloy 
```

Comment:
Are these valid links?

This is actually a case where shortcodes are *really* good: Hugo lets you write `{{% ref page %}}` for an internal page link, that then fails if the page does not exist. See the "case studies" ref on 18.

Reply:
not yet: i just dumped in old pages from the jekyll site for now.